Kpx_param: 2.0
Kpy_param: 1.1 #1.1
Kpy_param_rot: 2.0 #0.01
linear_vel: 0.1 #0.1
robot_width: 0.8
obs_coefficient: 0.5
front_obs: 0.8
boundary_percent: 0.02
spare_length: 1.5
amcl_driving: false
check_obstacles: true
rot_kx: 0.1
rot_ky: 0.05
rot_kt: 1.0
min_vel: 0.05 # The rover robotics can't move specific low speed
min_rot: 0.6
max_rot: 3.0
